### PR TITLE
MINOR: update jackson dependencies

### PR DESCRIPTION
**Pull request by mjsax**
*More detailed description of your change,
if necessary. The PR title and PR message become
the squashed commit message, so use a separate
comment to ping reviewers.*

*Summary of testing strategy (including rationale)
for the feature or bug fix. Unit and/or integration
tests are expected for any behaviour change and
system tests should be considered for larger changes.*

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: gradle/dependencies.gradle [hunk 1]
```diff
@@ -52,7 +52,8 @@ versions += [
   argparse4j: "0.7.0",
   bcpkix: "1.56",
   easymock: "3.4",
-  jackson: "2.8.5",
+  jackson: "2.8.11",
+  jacksonDB: "2.8.11.2",
```
Comment (ijuma) [inline gradle/dependencies.gradle]: We should just call it jacksonDatabind. DB is ambiguous and a bit confusing.

**Review by ijuma — APPROVED** [review]
LGTM, just a nit.

**Comment by mjsax** [comment]
Updated this.

**Review by ijuma — APPROVED** [review]
LGTM

**Pull request merged**